packages/parcel-transformer/src/types.ts

```diff
@@ -26,4 +27,9 @@ export interface ParcelTransformerOpts extends BabelPluginOpts {
    * Default to `true`.
    */
   optimizeCss?: boolean;
+
+  /**
+   * Override the default `resolve` passed into webpack, which is used to statically evaluate import declarations
```

Comment:
should reference parcel?

Reply:
should we also update the docs?

Reply:
Have created a PR to update the Website: https://github.com/compiled/website/pull/661